packages/flutter/lib/src/material/icons.dart

```diff
@@ -21,6 +21,27 @@ import 'package:flutter/widgets.dart';
 ///   uses-material-design: true
 /// ```
 ///
+/// {@tool snippet --template=stateless_widget_scaffold}
```

Comment:
[Nice-to-have] Adding a sample output image like [ListTile docs](https://master-api.flutter.dev/flutter/material/ListTile-class.html). See https://github.com/flutter/assets-for-api-docs for example pull requests. 

Let me know if you have any questions about how to get this set up!